api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+
+  if (req.headers['x-access-token']) {
+    const token = req.headers['x-access-token']
```

Comment:
If you move this before the `if` statement you wouldn't need to use `req.headers['x-access-token']` twice, you could just use your `token` constant